src/core/validateCommandOptions.js

```diff
@@ -13,27 +13,20 @@ governing permissions and limitations under the License.
 const COMMAND_DOC_URI = "https://adobe.ly/2UH0qO7";
 
 export default ({ command, options, logger }) => {
-  if (!command.optionsValidator) {
-    return;
-  }
-  let errors = [];
-  let warnings = [];
-  try {
-    const validationResult = command.optionsValidator(options);
-    errors = validationResult.errors || errors;
-    warnings = validationResult.warnings || warnings;
-  } catch (e) {
-    errors.push(e);
-  }
-  const validationProblems = errors.length > 0 ? errors : warnings;
-  const { commandName, documentationUri = COMMAND_DOC_URI } = command;
-  const invalidOptionsMessage = `Invalid ${commandName} command options:\n\t - ${validationProblems.join(
-    "\n\t - "
-  )}\nFor command documentation see: ${documentationUri}`;
-  if (errors.length) {
-    throw new Error(invalidOptionsMessage);
-  }
-  if (warnings.length) {
-    logger.warn(invalidOptionsMessage);
+  const {
+    commandName,
+    documentationUri = COMMAND_DOC_URI,
+    optionsValidator
+  } = command;
+  let validatedOptions = options;
+  if (optionsValidator) {
+    try {
+      validatedOptions = optionsValidator({ options, logger });
+    } catch (validationError) {
+      const invalidOptionsMessage = `Invalid ${commandName} command options:\n\t - ${validationError}
+      \nFor command documentation see: ${documentationUri}`;
```

Comment:
Because this is a template literal, I believe this will result in two line breaks after `validationError`. One would be due to the line break after `${validationError}` and another from `\n`. I would recommend putting the template literal on a single line:

```
const invalidOptionsMessage = `Invalid ${commandName} command options:\n\t - ${validationError}\nFor command documentation see: ${documentationUri}`;
```